exercises/zebra-puzzle/introduction.md

```diff
@@ -1,7 +1,7 @@
 # Introduction
 
 The Zebra Puzzle is a famous logic puzzle in which there are five houses, each painted a different color.
```

Comment:
I think it might help to state that this is a modified version of the Zebra Puzzle. A student who Googles "Zebra Puzzle" will likely find the previous version (or something similar along those lines). For example, the previous version are on [Wikipedia](https://en.wikipedia.org/wiki/Zebra_Puzzle) and [The Dinner Table](https://waitbutwhy.com/table/zebra-puzzle). 

Perhaps, something along the lines of:

> The Zebra Puzzle is a famous logic puzzle in which there are five houses.
> In this modified version, each house is painted a different color, ....

Reply:
The exercise does have a source URL and the name isn't changed. Are you concerned that students will be confused that this version is different from another version?

Reply:
Yeah, I was thinking it might be confusing for the student if they see another version, especially as the source URL points to Wikipedia, which describes the puzzle with the previous wording. Although the Wikiepdia page also does state that there are different versions of the puzzle, I think stating it will help to reduce that confusion.

Reply:
I think that students will see the versions differ if they compare the exercises. I don't think that needs pointing out. If they compare versions and notice they are different, I think most people will realize Exercism opted for different specifics for some reason. I don't especially think we need to spell that out in the instructions. 

I'm curious to hear what others think!

Reply:
I've seen many variations used when researching the exercise, so I think it's fine to have us use a variation.